runtime/karura/src/lib.rs

```diff
@@ -1926,6 +1926,7 @@ impl_runtime_apis! {
 			orml_add_benchmark!(params, batches, module_honzon, benchmarking::honzon);
 			orml_add_benchmark!(params, batches, module_cdp_treasury, benchmarking::cdp_treasury);
 			orml_add_benchmark!(params, batches, module_collator_selection, benchmarking::collator_selection);
+			// orml_add_benchmark!(params, batches, module_nominees_election, benchmarking::nominees_election);
```

Comment:
why commented out?

Reply:
karura-rumtime doesn't implement `module_nominees_election` runtime.

Reply:
oh ok. we can add enable it later